index.html

```diff
@@ -3,9 +3,34 @@
   <head>
     <meta charset="utf-8">
     <title>super-fantastic-dashboard</title>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <style>
+      body {
+        background-color: lightblue;
+        background-repeat: no-repeat;
+        background-position: center center;
+        background-attachment: fixed;
+        background-size: cover;
+      }
+    </style>
   </head>
   <body>
     <div id="app"></div>
     <script src="/dist/build.js"></script>
+    <script>
```

Comment:
I'm open to re-writing with a vanilla ajax request. I've not done that before, but the info is online. Good point, we're only using axios for this one very small thing. I'll take a look at doing that.